MaterialLib/Fluid/Density/LinearTemperatureDependentDensity.h

```diff
@@ -0,0 +1,80 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file:   LinearTemperatureDependentDensity.h
+ * \author: wenqing
+ *
+ * Created on August 10, 2016, 11:34 AM
+ */
+
+#ifndef LINEARTEMPERATUREDEPENDENTDENSITY_H
+#define LINEARTEMPERATUREDEPENDENTDENSITY_H
+
+#include <string>
+
+#include "BaseLib/ConfigTree.h"
+
+#include "MaterialLib/Fluid/FluidProperty.h"
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+/// Linear temperature dependent density model.
+class LinearTemperatureDependentDensity final : public FluidProperty
+{
+public:
+    /// \param config  ConfigTree object which contains the input data
+    ///                including <type>temperature_dependent</type> and it has
+    ///                a tag of <density>
+    explicit LinearTemperatureDependentDensity(BaseLib::ConfigTree const& config)
```

Comment:
Then I want to move this sort of parsing to createFluidDensityModel.h/cpp, which actually does parsing.